configs/slurm.config

```diff
@@ -12,38 +12,44 @@ executor {
 
 process {
 
-    time   = { check_max( 4.h  * task.attempt, 'time'   ) }
+    time = { 4.h  * task.attempt }
 
+    withLabel: 'process_single' {
+        cpus          = { 1                   }
+        memory        = { 6.GB * task.attempt }
+        errorStrategy = { task.exitStatus in ((130..155) + 104) ? 'retry' : 'finish' }
+        maxRetries    = 2
+    }
     withLabel: 'light' {
-        cpus = 1
-        memory = { 3.GB * task.attempt }
-        errorStrategy = { task.exitStatus == 137 ? 'retry' : 'finish' }
-        maxRetries = 2
+        cpus          = 1
+        memory        = { 3.GB * task.attempt }
+        errorStrategy = { task.exitStatus in ((130..155) + 104) ? 'retry' : 'finish' }
+        maxRetries    = 2
     }
     withLabel: 'medium' {
-        cpus = 8
-        memory = { 3.GB * task.attempt }
-        errorStrategy = { task.exitStatus == 137 ? 'retry' : 'finish' }
-        maxRetries = 2
+        cpus          = 8
+        memory        = { 3.GB * task.attempt }
+        errorStrategy = { task.exitStatus in ((130..155) + 104) ? 'retry' : 'finish' }
+        maxRetries    = 2
     }
     withLabel: 'heavy' {
-        cpus = 1
-        memory = { 10.GB + (10.GB * (task.attempt-1)) }
-        errorStrategy = { task.exitStatus == 137 ? 'retry' : 'finish' }
-        maxRetries = 2
+        cpus          = 1
+        memory        = { 10.GB + (10.GB * (task.attempt-1)) }
+        errorStrategy = { task.exitStatus in ((130..155) + 104) ? 'retry' : 'finish' }
```

Comment:
Can't you merge all of these with (DRY principle)?:
```
withLabel: 'medium|heavy|very_heavy' {
    errorStrategy = { task.exitStatus in ((130..155) + 104) ? 'retry' : 'finish' }
}
```

Reply:
Yes, that's true.
It looked like this before I added my branch. That's what I meant when I mentioned "messy configs".

To be honest, I would prefer to add up-to-date nf-core [`base.config`](https://github.com/nf-core/tools/blob/main/nf_core/pipeline-template/conf/base.config) (with Chris's values of mem and cpu ofc) and our [`slurm.config`](https://github.com/EBI-Metagenomics/amplicon-pipeline/blob/cb2c574af0fcd39666fb82709f165e110117bb51/conf/codon_slurm.config#L4) instead of this one "all-in_one" to make repo more standard. 

@mberacochea what do you think about this?

Reply:
yeah, go with Chris approach :)